backend/src/api/game/play.rs

```diff
@@ -0,0 +1,49 @@
+use crate::api::game::{game_request::GameRequest, game_response::GameResponse};
+use crate::server_error::ServerError;
+use actix_web::web::{self, Json, Path};
+use hive_lib::{
+    game_error::GameError, history::History, piece::Piece, position::Position, state::State,
+};
+
+fn get_game_state_from_db(_game_id: u64) -> Result<State, GameError> {
+    let game = "game.txt";
+    let history = History::from_filepath(game)?;
+    State::new_from_history(&history)
+}
+
+fn play_turn(mut state: State, piece: String, pos: String) -> Result<GameResponse, ServerError> {
+    let board_move = format!("{} {}", piece, pos);
+    let piece = Piece::from_string(&piece)?;
+    let pos = Position::from_string(&pos, &state.board)?;
+    state.play_turn(piece, pos)?;
+    let game = "game.txt";
+    state.history.write_move(game, state.turn, board_move);
```

Comment:
it'd be nice if all of the "fake backend" stuff lived behind a single interface, so we could cleanly rewrite it w/ real database calls all at once

Reply:
I want to get away from this asap, so I will remove all of this in my next PR and work on dieselifying everything